Example/Tests/BONTextAlignmentTestCase.m

```diff
@@ -75,6 +75,7 @@ - (void)testParagraphStyle
                           .headIndent(2.34)
                           .tailIndent(3.45)
                           .lineHeightMultiple(3.14)
+                          .lineBreakMode(NSLineBreakByTruncatingTail)
```

Comment:
Would you mind moving this down below `.lineSpacing(2.72)`? Line break mode is after line spacing everywhere else, and that consistency makes it very easy to validate that it's in all the places it should be.